hydromt_wflow/workflows/waterbodies.py

```diff
@@ -118,7 +118,7 @@ def waterbodymaps(
     ds_out["resareas"] = ds_out["resareas"].astype("float32")
 
     if wb_type == "lake":
-        ds_out = ds_out.rename({"resareas": "lakeareas", "reslocs": "lakelocs"})
+        ds_out = ds_out.rename({"resareas": "lake_area_id", "reslocs": "lakelocs"})
```

Comment:
I think it may have happened in other places but the following hydromt name should have been renamed in your PR:
```suggestion
        ds_out = ds_out.rename({"reservoir_area_id": "lake_area_id", "reservoir_outlet": "lake_outlet_id"})
```